resources/puppetlabs/puppetdb/benchmark/samples/facts/pg2.vm.json

```diff
@@ -9253,8 +9253,8 @@
             "gem"
         ],
         [
-            "puppetlabs-release-pc1",
-            "1.1.0-4jessie",
+            "puppet-release",
+            "1.0.0-1jessie",
```

Comment:
I have no idea what this does, but it references the old release packages. I assume you want the new one...